packages/volto/src/components/manage/Blocks/Video/View.test.jsx

```diff
@@ -20,29 +24,42 @@ config.blocks.blocksConfig = {
   },
 };
 
+const store = mockStore({
+  intl: {
+    locale: 'en',
+    messages: {},
+  },
+});
+
 test('renders a view video component', () => {
   const component = renderer.create(
-    <View
-      data={{
-        '@type': 'video',
-        url: 'https://youtu.be/KqjeO_ekW3g',
-      }}
-    />,
+    <Provider store={store}>
+      <View
+        data={{
+          '@type': 'video',
+          url: 'https://youtu.be/KqjeO_ekW3g',
+        }}
+      />
+      ,
```

Comment:
@Molochem by introducing this comma , you render it also in the test which results in a render of an array of a div and that comma instead of how it was previously, there is no need to have it as such and the test could be as it was with just the addition of the aria-label on the div